instrumentation/ratpack/ratpack-1.7/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("io.ratpack")
+    module.set("ratpack-core")
+    versions.set("[1.7.0,)")
+  }
+}
+
+dependencies {
+  library("io.ratpack:ratpack-core:1.7.0")
+
+  implementation(project(":instrumentation:netty:netty-4.1:library"))
+  implementation(project(":instrumentation:ratpack:ratpack-1.7:library"))
+
+  testLibrary("io.ratpack:ratpack-test:1.7.0")
```

Comment:
you can remove this, `library` already includes everything that the `testLibrary` does